index.js

```diff
@@ -181,19 +194,19 @@ class S3Adapter {
 
     const fileKey = `${this._bucketPrefix}${fileName}`;
 
-    let presignedUrl = '';
-    if (this._presignedUrl) {
-      const params = { Bucket: this._bucket, Key: fileKey };
-      if (this._presignedUrlExpires) {
-        params.Expires = this._presignedUrlExpires;
-      }
-      // Always use the "getObject" operation, and we recommend that you protect the URL
-      // appropriately: https://docs.aws.amazon.com/AmazonS3/latest/dev/ShareObjectPreSignedURL.html
-      presignedUrl = this._s3Client.getSignedUrl('getObject', params);
-      if (!this._baseUrl) {
-        return presignedUrl;
-      }
-    }
+    const presignedUrl = '';
+    // if (this._presignedUrl) {
+    //   const params = { Bucket: this._bucket, Key: fileKey };
+    //   if (this._presignedUrlExpires) {
+    //     params.Expires = this._presignedUrlExpires;
+    //   }
+    //   // Always use the "getObject" operation, and we recommend that you protect the URL
+    //   // appropriately: https://docs.aws.amazon.com/AmazonS3/latest/dev/ShareObjectPreSignedURL.html
+    //   presignedUrl = getSignedUrl(this._s3Client, 'getObject', params).then(result => );
+    //   if (!this._baseUrl) {
+    //     return presignedUrl;
+    //   }
+    // }
 
```

Comment:
I have disabled this test as I do not know how to properly migrate to AWS SDK v3 which does not offer a sync version of `getSignedUrl` anymore.